docs/local-module.md

```diff
@@ -0,0 +1,46 @@
+# Postman.gov.sg
+
+- [Background](#background)
+- [How to install](#how-to-install)
+- [Local development](#local-development)
+- [Build dependencies](#build-dependencies)
+- [More information](#more-information)
+
+## Background
+
+We wanted to find a way to share the templating logic because:
```

Comment:
I think it's a bit strange for someone new to the repo to suddenly see this sentence without context. Perhaps start by giving a brief explanation of what a local module is. Sharing the templating logic is just an example of a local module.